packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java

```diff
@@ -100,6 +102,24 @@ public static RestActionBodyExecTemplateModel createDefaultActionModel()
         return restActionModel;
     }
 
+    public static List<RestActionBodyExecTemplateModel> addActionContextParams(List<RestActionBodyExecTemplateModel> inputActions)
+    {
+        inputActions.forEach(inputAction -> {
+            final Map<String, Serializable> params = new HashMap<>((Map<String, Serializable>) inputAction.getParams());
+            params.put("actionContext", "rule");
```

Comment:
If this param is missing in default action, you could simply fix the `createDefaultActionModel()`

Reply:
This parameter is not supposed to be passed as input but is added during rule action validation (on create/update).
That is why I didn't add it in `createDefaultActionModel()` and created this method as it is and most likely will be used in multiple tests when we need to create expected rule with actions.